target_joint_topic_name: /target_joint_states
target_array_topic_name: /joint_impedance_controller/des_joint_pos
target_gripper_topic_name: /gripper_joint_controller/command
